Universidad/dominio/dominio/IAgregarAlumno.java

```diff
@@ -0,0 +1,8 @@
+package dominio;
+
+import java.util.Set;
+
+public interface IAgregarAlumno {
+	boolean addAlumnos(Alumno alumno, Set<Alumno>alumnosUniversidad);
```

Comment:
En esta definición estamos generando una dependencia directa de nuestro "repositorio" o "base de datos". La funcionalidad no debe saber en dónde guardar la data, y la invocación al método no debería en cada llamada indicarle donde almacenar (salvo caso puntual)

```suggestion
	boolean addAlumno(Alumno alumno);
```